### PR TITLE
Add ability to show image on index

### PR DESCRIPTION
**Pull request by jondkinney**

### CODE REVIEW

**Comment by michelegera** [comment]
Thanks for your PR, looks great!

Could you please add a tiny bit of documentation for the `image_on_index` option in the appropriate section of the README?

Then I’ll merge and cut a new release of the gem 💎.

**Comment by jondkinney** [comment]
@michelegera added some info to the readme and rebased/squashed it up for you. LMK if you need anything else. 

One possible enhancement would be to reference a `thumb` version of the image, if available, to save on load time for the index. But I think that's could also be added as an enhancement after merging this in.

File: app/views/fields/carrierwave/_show.html.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@ for this attribute
     <span>No file(s) uploaded yet</span>
   <% end %>
 <% else %>
-  <% if field.image.present? %>
+  <% if field.image.present? && !field.data.model.new_record? %>
```
Comment (michelegera) [inline app/views/fields/carrierwave/_show.html.erb]: Can you please amend the second half of this to `field.data.model.persisted?`?

I think positive conditions reads better than negatives!

Comment (jondkinney) [reply app/views/fields/carrierwave/_show.html.erb]: Agreed! Tweak incoming.

**Comment by jondkinney** [comment]
@michelegera should be all set.

**Comment by michelegera** [comment]
@jondkinney thanks for your work on this! I’m going to merge and release a new version of the gem.

Definitely agree with you on the `thumb` to display on index for increased performance. Would you mind submitting a PR for that, when you have time?

**Pull request merged**